src/main/rx/crsf.c

```diff
@@ -172,7 +172,8 @@ static void handleCrsfLinkStatisticsFrame(const crsfLinkStatistics_t* statsPtr,
 
 #ifdef USE_RX_LINK_QUALITY_INFO
     if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) {
-        setLinkQualityDirect(scaleCrsfLq((stats.rf_Mode * 100) + stats.uplink_Link_quality));
+        setLinkQualityDirect(scaleCrsfLq(stats.uplink_Link_quality));
```

Comment:
We need to fix this unnecessary * 3.41 that's in the code for some reason. The link quality is getting multiplied by 3.41 and then everywhere we use it it's divided by 3.41. According to the CRSF spec the link quality is an integer from 0-99. We just need to use that value without any calculations applied to it.

Reply:
Ok, I wasn't sure why that factor was even in there in the first place